gulpfile.js

```diff
@@ -87,7 +88,7 @@ gulp
   })
   .task('validate:js', () => {
     return gulp
-      .src(javascriptFilePaths)
+      .src(fileArgs.file? fileArgs.file : javascriptFilePaths)
```

Comment:
nit pick - can you add a space between "file" and "?" in these ternary statements?

Reply:
Updated!